legate/driver/args.py

```diff
@@ -356,20 +356,48 @@
     dest="bind_detail",
     action="store_true",
     required=False,
-    help="print out the final invocation run by bind.sh, "
+    help="print out the final invocation run by bind.sh "
     "[legate-only, not supported with standard Python invocation]",
 )
 
 
 other = parser.add_argument_group("Other options")
 
+other.add_argument(
+    "--wrapper",
+    dest="wrapper",
+    required=False,
+    action="append",
+    default=[],
+    help="Specify another executable (and any command-line arguments for that "
+    "executable) that will be responsible to indirectly invoke legate, "
+    "including any other wrappers. May contain the special string "
```

Comment:
```suggestion
    "executable) to wrap the Legate executable invocation. This wrapper will "
    "come right after the launcher invocation, and will be passed the rest of "
    "the Legate invocation (including any other wrappers) to execute. May "
    "contain the special string "
```

suggested rewording, similar for `--wrapper-inner`

Reply:
https://github.com/nv-legate/legate.core/pull/813/commits/dc345d6ed0a428c5e786b84efa836b52a87f92c7